Library/Formula/wireshark.rb

```diff
@@ -3,13 +3,9 @@ class Wireshark < Formula
   homepage "https://www.wireshark.org"
 
   stable do
-    url "https://www.wireshark.org/download/src/all-versions/wireshark-1.12.8.tar.bz2"
-    mirror "https://1.eu.dl.wireshark.org/src/wireshark-1.12.8.tar.bz2"
-    sha256 "357e0a4e49525d80cdc740bb16539fcdb526ad38cc2ed6cabedafc9bdee5c7e7"
-
-    # Removes SDK checks that prevent the build from working on CLT-only systems
-    # Reported upstream: https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=9290
-    patch :DATA
+    url "https://www.wireshark.org/download/src/all-versions/wireshark-2.0.0.tar.bz2"
```

Comment:
In [the console text](http://bot.brew.sh/job/Homebrew%20Pull%20Requests/37916/version=mavericks/consoleText) of failed build, we can see

```
checking whether we can build for OS X 10.9... yes, with the 10.10 SDK
```

Should it point to 10.9 SDK?

```zsh
# On 10.9.5 with Xcode 6.2 and CLT 6.2.0.0.1.1424975374

$ sw_vers -productVersion
10.9.5
$ xcrun --show-sdk-version
10.10
```

libpcap 1.5.x in 10.10 SDK has the symbol ```PCAP_TSTAMP_PRECISION_NANO``` while libpcap 1.3.0 in 10.9 SDK doesn't.